composer.json

```diff
@@ -23,16 +23,22 @@
     ],
     "require": {
         "php": "^7.0",
+        "illuminate/pagination": "~5.5.0",
         "illuminate/support": "~5.5.0",
         "illuminate/validation": "~5.5.0",
-        "illuminate/pagination": "~5.5.0",
         "illuminate/view": "~5.5.0"
     },
     "require-dev": {
+        "amphp/amp": "^2.0.3",
```

Comment:
You can drop that one now, as `amphp/parallel-functions@0.1.2` enforces that.

Reply:
Will do